phpBB/install/schemas/schema_data.sql

```diff
@@ -155,6 +155,8 @@ INSERT INTO phpbb_config (config_name, config_value) VALUES ('img_max_height', '
 INSERT INTO phpbb_config (config_name, config_value) VALUES ('img_max_thumb_width', '400');
 INSERT INTO phpbb_config (config_name, config_value) VALUES ('img_max_width', '0');
 INSERT INTO phpbb_config (config_name, config_value) VALUES ('img_min_thumb_filesize', '12000');
+INSERT INTO phpbb_config (config_name, config_value) VALUES ('img_quality', '90');
+INSERT INTO phpbb_config (config_name, config_value) VALUES ('img_strip_metadata', '1');
```

Comment:
Should be IMHO false per default.

Reply:
I agree with @3D-I as this would be in line with current behavior and might otherwise come as a surprise to users of phpBB.

Reply:
I personally think that most users of phpBB would be surprised that the default behaviour has been to preserve personally-identifiable metadata as this has never been disclosed to administrators or end-users. However, I understand your reasoning and will amend.